src/Util/HTTPSignature.php

```diff
@@ -408,7 +409,7 @@ public static function fetch($request, $uid)
 	 *                         'nobody' => only return the header
 	 *                         'cookiejar' => path to cookie jar file
 	 *
-	 * @return object CurlResult
+	 * @return CurlResult|void CurlResult
```

Comment:
This shouldn't return `void` but throw an Exception if the `$owner` variable is empty, but maybe it's outside of the scope of this PR.

Reply:
I reverted it .. it's really part of another PR

The reason for it: https://github.com/friendica/friendica/blob/218d451248c585b9c97915d852d4bacbff838ecf/src/Util/HTTPSignature.php#L422 and https://github.com/friendica/friendica/blob/218d451248c585b9c97915d852d4bacbff838ecf/src/Util/HTTPSignature.php#L427